ledger/participant-integration-api/src/main/scala/platform/store/dao/events/EventsTableFlatEvents.scala

```diff
@@ -88,24 +88,24 @@ private[events] object EventsTableFlatEvents {
       "create_key_value_compression",
     ).mkString(", ")
 
-  private val groupByColumns =
-    Seq(
-      "event_offset",
-      "transaction_id",
-      "ledger_effective_time",
-      "command_id",
-      "workflow_id",
-      "participant_events.event_id",
-      "contract_id",
-      "template_id",
-      "create_argument",
-      "create_argument_compression",
-      "create_signatories",
-      "create_observers",
-      "create_agreement_text",
-      "create_key_value",
-      "create_key_value_compression",
-    ).mkString(", ")
+//  private val groupByColumns =
```

Comment:
Please restore the group by for non-oracle.

Reply:
restored for non-oracle, kept out for oracle